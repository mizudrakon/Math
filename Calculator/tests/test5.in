11
-3
123 + 3 * 2
x=20 / (10-2)
y=x
y=$x
y=$x * $x 
xyz=         -(1 + 2 - ((-3) * (2)))
wdisi  sususus 132
$X * $Y 
 z=3
zz= $xyz * $z
