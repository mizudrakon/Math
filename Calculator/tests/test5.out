-3
129
2
invalid
2
4
-9
invalid
8
3
-27
